active/online-community.md

```diff
@@ -0,0 +1,97 @@
+# Online Community Working Group
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The [Django Forum](https://forum.djangoproject.com/) and the [Django Discord](https://discord.gg/xcRH6mN4fa) are core parts of the overall community and those that help and moderate these platforms take significant time from their day to help others.
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - facilitate and centralise discussion between moderators and admins on the various platforms
+  - coordinate efforts for training and processes around moderation
+  - provide support to improve the platforms to lighten moderation and admin workloads
+  - provide training and support to the wider community on how to use the platforms effectively
+  - ensure consistency that our online Django communities are welcoming and safe spaces
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms. Examples of these processes include
+    - Proposals for adding/removing moderators & other roles and the adding/removing of roles directly
+    - Training for moderators & other roles as required
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+
+- Which actions will the WG take back to the Board for votes?
+  - Proposal to trial a new platform
+  - Proposal to migrate to a new platform
+  - Proposal to retire an existing platform that is in use
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- Administrator Privileges on approved community platforms with the following conditions to create redundancy and protect the community from any single person holding too much power:
+  - A single member may not have Administrator Privileges to all platforms
+  - Each platform has a minimum of two members with Administrator Privileges
+
+## Initial membership
+
+- Chair: Andrew Miller
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Tom Carrick
+- Other members:
+  - Patryk Bratkowski
+  - Natalia Bidart
+  - Ben Cardy
+  - (CoC Liason)
+  - (SC Liason?)
```

Comment:
I suggest these are also filled before voting.

I wouldn't consider the SC blocking at this moment.

Reply:
If those are intended formal roles within the group, I think it’d be nice to document what the roles involve in this document. For "board liaison" for example, people’s expectations have differed on quite fundamental aspects. For example whether the named board liaison is a member of a group in their own right, or more of an "observer" that doesn’t necessarily vote or take part in group activities aside from interactions with the board.